azurerm/resource_arm_maps_account.go

```diff
@@ -0,0 +1,186 @@
+
+	if requireResourcesToBeImported && d.IsNewResource() {
+		existing, err := client.Get(ctx, resGroup, name)
+		if err != nil {
+			if !utils.ResponseWasNotFound(existing.Response) {
+				return fmt.Errorf("Error checking for presence of existing Maps Account %q (Resource Group %q): %+v", name, resGroup, err)
+			}
+		}
+
+		if existing.ID != nil && *existing.ID != "" {
+			return tf.ImportAsExistsError("azurerm_maps_account", *existing.ID)
+		}
+	}
+
+	parameters := maps.AccountCreateParameters{
+		Location: &location,
+		Sku: &maps.Sku{
+			Name: &sku,
+		},
+		Tags: expandTags(tags),
+	}
+
+	if _, err := client.CreateOrUpdate(ctx, resGroup, name, parameters); err != nil {
+		return fmt.Errorf("Error creating/updating Maps Account %q (Resource Group %q) %+v", name, resGroup, err)
+	}
+
+	read, err := client.Get(ctx, resGroup, name)
+	if err != nil {
+		return fmt.Errorf("Error retrieving Maps Account %q (Resource Group %q) %+v", name, resGroup, err)
+	}
+
+	if read.ID == nil {
+		return fmt.Errorf("Cannot read Maps Account %q (Resource Group %q) ID", name, resGroup)
+	}
+
+	d.SetId(*read.ID)
+
+	return resourceArmMapsAccountRead(d, meta)
+}
+
+func resourceArmMapsAccountRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).maps.AccountsClient
+	ctx := meta.(*ArmClient).StopContext
+
+	id, err := parseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+	resGroup := id.ResourceGroup
+	name := id.Path["accounts"]
+
+	resp, err := client.Get(ctx, resGroup, name)
+	if err != nil {
+		if utils.ResponseWasNotFound(resp.Response) {
+			d.SetId("")
+			return nil
+		}
+
+		return fmt.Errorf("Error making Read request on Maps Account %q (Resource Group %q): %+v", name, resGroup, err)
+	}
+
+	d.Set("name", resp.Name)
+	d.Set("resource_group_name", resGroup)
+	if sku := resp.Sku; sku != nil {
+		d.Set("sku", string(*sku.Name))
+	}
+	if props := resp.Properties; props != nil {
+		d.Set("x_ms_client_id", props.XMsClientID)
+	}
+
+	keysResp, err := client.ListKeys(ctx, resGroup, name)
+	if err != nil {
+		if utils.ResponseWasNotFound(resp.Response) {
+			d.SetId("")
+			return nil
+		}
+		return fmt.Errorf("Error making Read Access Keys request on Maps Account %q (Resource Group %q): %+v", name, resGroup, err)
+	}
+	d.Set("primary_access_key", keysResp.PrimaryKey)
+	d.Set("secondary_access_key", keysResp.SecondaryKey)
+
+	flattenAndSetTags(d, resp.Tags)
+
+	return nil
+}
+
+func resourceArmMapsAccountDelete(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).maps.AccountsClient
+	ctx := meta.(*ArmClient).StopContext
+
+	id, err := parseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+	resGroup := id.ResourceGroup
+	name := id.Path["accounts"]
+
+	if _, err := client.Delete(ctx, resGroup, name); err != nil {
+		return fmt.Errorf("Error deleting Maps Account %q (Resource Group %q): %+v", name, resGroup, err)
+	}
+
+	return nil
+}
+
+func globalLocation() string {
```

Comment:
Could we hard code this instead of having a function?

Reply:
Is there anywhere the Terraform provider keeps constants like this?